cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/export/elements/StaticVarCompensatorEq.java

```diff
@@ -51,7 +51,9 @@ public static void write(String id, String svcName, String equipmentContainer, S
         }
         writer.writeEndElement();
         writer.writeEmptyElement(cimNamespace, EQ_STATICVARCOMPENSATOR_SVCCONTROLMODE);
-        writer.writeAttribute(RDF_NAMESPACE, CgmesNames.RESOURCE, cimNamespace + regulationMode(svcControlMode));
+        if (svcControlMode != null) {
+            writer.writeAttribute(RDF_NAMESPACE, CgmesNames.RESOURCE, cimNamespace + regulationMode(svcControlMode));
+        }
```

Comment:
```suggestion
    writer.writeAttribute(RDF_NAMESPACE, CgmesNames.RESOURCE, cimNamespace + regulationMode(svcControlMode));
```
Nomrally `svcControlMode` can not be null